Sources/Kinect/Microsoft.Psi.AzureKinect.x64/AzureKinectSensorConfiguration.cs

```diff
@@ -11,6 +11,22 @@ namespace Microsoft.Psi.AzureKinect
     /// </summary>
     public class AzureKinectSensorConfiguration
     {
+        /// <summary>
+        /// Enum for powerline frequency.
+        /// </summary>
+        public enum PowerlineFrequencyTypes
+        {

```

Comment:
Maybe there should be an additional `Default = 0` value here, which means  we don't call `this.device.SetColorControl` with an explicit `PowerlineFrequency` in `AzureKinectCore` and defer to whatever the core library defaults to.

Reply:
I added a default and only sets the powerfrequency if its not the default. The Azure SDK only takes 1 or 2 and will throw an exception if zero is passed in.